src/components/MainFooter.vue

```diff
@@ -1,5 +1,17 @@
 <template>
-    <footer>
-        <h1>This is a footer</h1>
+  <div id="bot">
```

Comment:
This seems to break https://github.com/fspexet/fspexet-vue-ts/blob/9be45e7964183732e2809c3bb693438ea194e413/src/app.scss#L9-L12
Is there a reason for this and if there is then that functionality should be applied to it as well.

Reply:
Makes no sense and replaced with things that actually has meaning.